components/thumbnail/list/src/index.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import ThumbnailBasic from '@schibstedspain/sui-thumbnail-basic'
+
+const ThumbnailList = ({items, captionText, placeholder}) =>
+  <ul className='sui-ThumbnailList'>
+    {
+      items.map((item, index) =>
+        <li className='sui-ThumbnailList-item' key={index}>
+          <ThumbnailBasic
+            image={item.image}
+            link={item.link}
+            fallbackImage={item.fallbackImage}
+            captionText={captionText}
+            placeholder={placeholder}
+          />
+        </li>
+      )
+    }
+  </ul>
+
+ThumbnailList.displayName = 'ThumbnailList'
+
+ThumbnailList.propTypes = {
+  /**
+   * array of thumbnail/basic images
+   */
+  items: PropTypes.any.isRequired,
+  /**
+   * see thumbnail/basic for description
+   */
+  captionText: PropTypes.string.isRequired,
+  /**
+   * see thumbnail/basic for description
+   */
+  placeholder: PropTypes.node.isRequired
+}
```

Comment:
Los comentarios son necesarios?

Reply:
You can use @link or even import the proptype from ThumbnailBasic.proptypes (just a proposal)

Reply:
Using @link creates a link to github.com/link and I would like to avoid importing propTypes as I still believe in a prod env without propTypes

Reply:
I meant @link as JSDoc param i ! http://usejsdoc.org/tags-inline-link.html
You can even use http://usejsdoc.org/tags-see.html
Sorry for the misunderstanding LOL

Reply:
It's what I  tried, @link creates a link to github.com/link and @see creates a link to github.com/see

Reply:
OMG let's drop it XD